packages/web-components/src/components/masthead/masthead-logo.ts

```diff
@@ -22,7 +26,19 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  * @element dds-masthead-logo
  */
 @customElement(`${ddsPrefix}-masthead-logo`)
-class DDSMastheadLogo extends FocusMixin(LitElement) {
+class DDSMastheadLogo extends FocusMixin(HostListenerMixin(LitElement)) {
+  /**
+   * Media query listener handler.
+   */
+  private _mqHandler;
```

Comment:
```suggestion
  private _mqHandler: Handle | null = null;
```

Also please add `import Handle from '../../globals/internal/handle';`.